crates/feldera-types/src/error.rs

```diff
@@ -11,14 +11,14 @@ pub const MAX_REPORTED_PARSE_ERRORS: usize = 1_000;
 #[derive(Clone, Debug, Serialize, Deserialize, PartialEq, Eq, ToSchema)]
 pub struct ErrorResponse {
     /// Human-readable error message.
-    #[schema(example = "Unknown input format 'xml'.")]
+    #[schema(example = "Explanation of the error that occurred.")]
```

Comment:
Shouldn't `example` be an example?

Reply:
This is a special case unfortunately because if response type is `ErrorResponse` it will populate for every HTTP error code that does not have an example, provide as example that XML was not a valid input format, irrespective of whether it is even possible to supply XML for that endpoint, or if it is even applicable to the error code (in particular for the 500). This is rather confusing in the auto-generated docs. As such, I replaced it with this pseudo-example.